underfs/hdfs/src/main/java/tachyon/underfs/hdfs/HdfsUnderFileSystem.java

```diff
@@ -121,15 +124,21 @@ public void close() throws IOException {
   @Override
   public FSDataOutputStream create(String path) throws IOException {
     IOException te = null;
-    int cnt = 0;
-    while (cnt < MAX_TRY) {
+    RetryPolicy retryPolicy = new SleepingRetry(MAX_TRY) {
```

Comment:
Implementing `RetryPolicy` makes the intention easier to understand and avoids calls to `sleep(0)`. The code for `CountingRetry` should be really simple.